example/bootstrap_androidbp.bash

```diff
@@ -50,13 +51,18 @@ EOF
 }
 
 MENU=0
-PARAMS=$(getopt -o hm -l help,menuconfig --name $(basename "$0") -- "$@")
+PARAMS=$(getopt -o c:hm -l configdir,help,menuconfig --name $(basename "$0") -- "$@")
```

Comment:
Since we have "c:", this takes an option. configdir should also take an option, so should be "configdir:"

This applies to the other bootstrap files in this review too